intro-to-rails-2-relationships/nested-routes.md

```diff
@@ -89,12 +177,50 @@ class BooksController < ApplicationController
 end
 ```
 
-<details>
-  <summary>
-    <strong>Question:</strong> What should our code do if the author is not found, that is, if the user goes to <code style="white-space: nowrap;">/authors/789012/books</code> or <code>/authors/toaster/books</code>?
-  </summary>
-  Though the above code does not handle this case, if we were doing this for a project, we would want to make sure that we respond with a not_found (404) response code in this case.
-</details>
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: multiple-choice
+* id: 09f3c6d2-78c2-4e91-b1de-ca422a7538a0
+* title: What about if the author isn't found?
+* points: 1
+* topics: rails, rails-routes
+
+##### !question
+
+<strong>Question:</strong> What should our code do if the author is not found, that is, if the user goes to <code style="white-space: nowrap;">/authors/789012/books</code> or <code>/authors/toaster/books</code>?
```

Comment:
When I skimming this I read it as "would" so I just thought it'd be worth adding some emphasis here.
```suggestion
<strong>Question:</strong> What <em>should</em> our code do if the author is not found, that is, if the user goes to <code style="white-space: nowrap;">/authors/789012/books</code> or <code>/authors/toaster/books</code>?
```